sentry-sidekiq/lib/sentry/sidekiq/error_handler.rb

```diff
@@ -40,7 +42,13 @@ def retry_limit(context)
         when Integer
           limit
         when TrueClass
-          ::Sidekiq.options[:max_retries] || 25
+          max_retries =
+            if WITH_SIDEKIQ_7
+              ::Sidekiq.default_configuration[:max_retries]
```

Comment:
is `default_configuration` the right thing if the user overrides `max_retries`?

Reply:
From [the only invocation of `max_retries`](https://github.com/mperham/sidekiq/blob/ad9442f116086424948d60ec059755f9168a6bc8/lib/sidekiq/job_retry.rb#L73), I think users need to either assign the `default_configuration` value, or configure it at job level (which is handled in the `Integer` case).

So yeah, in this condition `default_configuration` is the right place to check.